src/tasky/__main__.py

```diff
@@ -1,86 +1,86 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 
 """
 A Google Tasks command line interface.
 """
 
 __author__ = 'Ajay Roopakalu (https://github.com/jrupac/tasky)'
 
-import codecs
 import datetime as dt
-import httplib2
 import os
 import shlex
 import sys
 import time
+import json
 
-import gflags
+from absl import flags
 
 from collections import OrderedDict
 
-from apiclient.discovery import build
-from oauth2client.client import OAuth2WebServerFlow
-from oauth2client.file import Storage
-from oauth2client.tools import run_flow as run
+from googleapiclient.discovery import build
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
 
+# Environment constants
+TASKY_DIR = os.path.join(os.environ['HOME'], '.tasky')
+KEYS_FILE = os.path.join(TASKY_DIR, 'client_id.json')
+TOKEN_FILE = os.path.join(TASKY_DIR, 'user_token.json')
+SCOPES = ['https://www.googleapis.com/auth/tasks']
```

Comment:
Since `SCOPES` is used in multiple files, maybe it's better to have one place where it's defined and referenced throughout?